bin/anthology/papers.py

```diff
@@ -140,8 +139,6 @@ def from_xml(xml_element, *args):
         # Set values from parsing the XML element (overwriting
         # and changing some initialized from the volume metadata)
         for key, value in parse_element(xml_element).items():
```

Comment:
I remember understanding all this logic at some point. It seems simpler now, but still cryptic what's going on. Might be worth documenting in comments.

Reply:
The new library should be much clearer in this regard, so I wouldn't worry about documenting the old one tbh.